src/Parser/Core/Modules/Haste.js

```diff
@@ -37,6 +37,8 @@ class Haste extends Analyzer {
     [SPELLS.ICY_VEINS.id]: 0.3,
     [SPELLS.BONE_SHIELD.id]: 0.1, // Blood BK haste buff from maintaining boneshield
     ...HIGH_TOLERANCE_HASTE_FNS,
+    [SPELLS.METAMORPHOSIS_HAVOC_BUFF.id]: 25,
+    [SPELLS.HAVOC_T21_4PC_BUFF.id]: 25,
```

Comment:
Shouldn't these be `.25` instead of `25`?  I think `25` would get read as 2500% haste

Reply:
fixed